sdk/keyvault/Azure.Security.KeyVault.Keys/src/RsaKeyCreateOptions.cs

```diff
@@ -12,22 +12,23 @@ namespace Azure.Security.KeyVault.Keys
 {
     public class RsaKeyCreateOptions : KeyCreateOptions
     {
-        public int KeySize { get; set; }
+        public string Name { get; set; }
         public KeyType KeyType { get; set; }
+        public int KeySize { get; set; }
+        public bool Hsm { get; set; }
 
-        public RsaKeyCreateOptions() 
+        public RsaKeyCreateOptions(bool hsm) 
         {
-            KeyType = KeyType.Rsa;
-        }
+            Name = Name;
```

Comment:
>Name = Name; [](start = 12, length = 12)

assigning the property to the property :)

Reply:
lol. Thanks!